libs/packages/layouts/src/lib/feature/side-toolbar/side-toolbar.component.html

```diff
@@ -7,4 +7,24 @@
     (click)="onResponsiveViewButtonClicked()" [attr.aria-label]="responsiveButtonText">
     {{responsiveButtonText}}
   </button>
-</div>
\ No newline at end of file
+</div>
+
+<ng-template #mobile>
+  <div class="sds-card">
+    <div class="sds-card__header sds-card__header--accent-cool grid-row">
+      <button id="cancelButton" class="sds-button sds-button--circular sds-button--white" (click)="onCancelClicked()"
+        [attr.aria-label]="backButtonAria">
+        <usa-icon [icon]="'chevron-left'"></usa-icon>
+      </button>
+      <div class="margin-x-auto">
+        <h2 class="sds-card__title margin-top-1 margin-left-neg-2">{{dialogTitleText}}</h2>
+      </div>
+    </div>
+  </div>
+  <ng-container *ngTemplateOutlet="template"></ng-container>
+  <div class="sds-card" *ngIf="showApply">
+    <div class="sds-card__body sds-card__body--accent-cool">
+      <button class="usa-button" id="applyButton" aria-label="Apply" (click)="onApplyClicked()">Apply</button>
```

Comment:
Since this has been moved to common component side, let's update the id to be less generic due to potential id conflicts. Tying them to responsiveButtonText variable is one suggestion - `${responsiveButtonText}applyButton` - similar for cancel button